mousestyles/path_diversity/path_features_advanced_proposed.py

```diff
@@ -0,0 +1,71 @@
+from future import print_function, absolute_import, division
+import numpy as np
+import pandas as pd
+from mousestyles.path_diversity.path_features import angle_between
```

Comment:
Yeah most of these imports were just me testing input at a different place in my ipynb. This wasn't meant to be a final version of anything, it's an exploration file which is why the imports are different.